test/grouping.jl

```diff
@@ -1585,15 +1607,8 @@ end
     rename!(df, [:d, :e, :f])
 
     @test names(gd) == names(df)
-    @test groupcols(gd) == [:d, :e]
-    @test valuecols(gd) == [:f]
-    @test map(NamedTuple, keys(gd)) ≅
```

Comment:
no - `keys(gd)` creates `GroupKeys` object with `GroupedDataFrame` as the only field, so it does nothing itself. However, when trying to e.g. show it an error will be thrown as the columns it looks for do not exist. Still `keys(gd)` will work, simply trying to use it will lead to errors.

Reply:
and it is tested above for a valid scenario

Reply:
But isn't it worth checking that `map(NamedTuple, keys(gd))` gives an error?

Reply:
OK - added